src/pages/starter-kit/checkout/observability.md

```diff
@@ -0,0 +1,176 @@
+
+#### Viewing Telemetry Data
+
+By default, diagnostics logging is disabled in the starter kit (`diagnostics: false` in [[`actions/telemetry.js`](https://github.com/adobe/commerce-checkout-starter-kit/blob/main/actions/telemetry.js)]). This setting only suppresses internal OpenTelemetry diagnostic output (SDK self‑logs). It does not control or forward your actual telemetry signals (logs, traces, metrics). Signals remain local until you configure exporters.
+
+**Diagnostics configuration (optional):** Provide an object instead of `false` to enable and tune internal OpenTelemetry logs. Example:
+
+```javascript
+{
+  // It's an object, this will enable OpenTelemetry internal logs
+  diagnostics: {
+    // Will only show warning logs and above
+    logLevel: "warning",
+    // The internal logs of OpenTelemetry will not get exported, only printed.
+    // This means you won't see them in your observability backend, but you will
+    // in the Adobe I/O Runtime logs (queryable via `aio rt activation logs <id>`)
+    exportLogs: false
+  }
+}
+```
+
+Set back to `false` to silence these internal logs:
+
+```javascript
+diagnostics: false
+```
+
+##### Exporting telemetry data (cloud or local)
+
+To forward telemetry (logs, traces, metrics) to any OTLP‑compatible service (collector, Grafana, DataDog, New Relic, etc.), you only need to adjust two places:
+
+1. [`actions/telemetry.js`](https://github.com/adobe/commerce-checkout-starter-kit/blob/main/actions/telemetry.js) – add exporter wiring
+2. [`app.config.yaml`](https://github.com/adobe/commerce-checkout-starter-kit/blob/main/app.config.yaml) – surface endpoint / auth values as action inputs (or use secrets)
+
+Template ([`actions/telemetry.js`](https://github.com/adobe/commerce-checkout-starter-kit/blob/main/actions/telemetry.js)):
+
+```javascript
+// actions/telemetry.js (template)
+
+import {
+  defineTelemetryConfig,
+  getAioRuntimeResource,
+  getPresetInstrumentations,
+  // localCollectorConfig, // optional helper if you want a local collector
+  // other helpers from '@adobe/aio-lib-telemetry' as needed
+} from '@adobe/aio-lib-telemetry';
+
+// Exporter wiring helper (keep minimal – replace stubs with real exporters per use case docs)
+function buildExporters(params) {
+  return {
+    // Logs: add your log record processor + exporter instance here
+    // logRecordProcessors: params.OTLP_ENDPOINT ? [ buildYourLogExporter(params) ] : [],
+
+    // Traces: uncomment and provide span processor/exporter if you enable tracing
+    // spanProcessors: params.OTLP_TRACES_ENDPOINT ? [ buildYourTraceExporter(params) ] : [],
+
+    // Metrics: provide metrics exporters if metrics collection is enabled
+    // metricsExporters: [],
```

Comment:
Is there any reason the codes are commented out? Could we just enable it, or is there any specific reason behind this demonstration?